### DISCUSSION TITLE
control from phone

### DISCUSSION THREAD

**Discussion by ks2hot4u** [AndreyPavlenko/Fermata#258] [Ideas]
As I own a Mazda 3 and all android auto actions are controlled by knob and buttons with no touchscreen, having the ability to use my phone for keyboard input or selecting/scrolling would be a welcome addition. I had hoped this would be possible by way of fermata control or the appstore version but no luck.

**Comment by fwon2638** [comment]
It can now be controlled via the rotary dial. I have a Mazda 6. Check out the video (and the description)

https://www.youtube.com/watch?v=X6qklvaVofw

**Comment by Alejandro2454** [comment]
Hello dear, excellent information given that I also have a 2019 Mazda 6. A
perhaps silly question, in point 2, what does "nx" mean?
